### PR TITLE
Add AdditionalInfo component analytics

### PR DESCRIPTION
**Pull request by bsmartin-ep**
## Description

This PR will:

1. Update component-library package version
2. Add AdditionalInfo expand / contract taxonomy to lookup object

https://github.com/department-of-veterans-affairs/va.gov-team/issues/14488
https://github.com/department-of-veterans-affairs/component-library/pull/40

## Acceptance criteria
- [x] No existing taxonomy on this one, so we just need to make sure it's firing correctly wherever this component is used.

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/platform/site-wide/component-library-analytics-setup.js [hunk 1]
```diff
@@ -7,6 +7,10 @@ import { kebabCase } from 'lodash';
 
 const analyticsEvents = {
   Modal: [{ action: 'show', event: 'int-modal-click' }],
+  AdditionalInfo: [
+    { action: 'expand', event: 'int-additional-info-expand' },
+    { action: 'contract', event: 'int-additional-info-contract' },
```
Comment (ncksllvn) [inline src/platform/site-wide/component-library-analytics-setup.js:12]: `contract`, not `collapse`?

We don't need to worry about this at all? `src/applications/static-pages/subscribeAdditionalInfoEvents.js`

**Review by timwright12 — APPROVED** [review]

File: src/platform/site-wide/component-library-analytics-setup.js (hunk 1 above)
Comment (bsmartin-ep) [reply src/platform/site-wide/component-library-analytics-setup.js]: > `contract`, not `collapse`?

Indeed. Fixed.

> We don't need to worry about this at all? `src/applications/static-pages/subscribeAdditionalInfoEvents.js`

Interesting!  I did not know there was analytics events in the formation package.  Correct me if I'm wrong, but this isn't using the actual component-library `AdditionalInfo` component, but more of a static site / Drupal recreation?

**Review by jonwehausen — APPROVED** [review]
Looks good!

File: src/platform/site-wide/component-library-analytics-setup.js (hunk 1 above)
Comment (bsmartin-ep) [reply src/platform/site-wide/component-library-analytics-setup.js]: @ncksllvn -

I took a look at one of the pages recording the formation events you linked (`/health-care/how-to-apply/index.html`) and it looks like it's separate.  I just see the formation events and not my new component library events...

http://35de3e34a2412cc9162c460c59689be8.review.vetsgov-internal/health-care/how-to-apply/index.html

```
monitorEvents(document.body, "component-library-analytics");
monitorEvents(document.body, '@department-of-veterans-affairs/formation/additional-info/expand');
monitorEvents(document.body, '@department-of-veterans-affairs/formation/additional-info/collapse');
```

image

**Review by ncksllvn — APPROVED** [review]

File: src/platform/site-wide/component-library-analytics-setup.js (hunk 1 above)
Comment (ncksllvn) [reply src/platform/site-wide/component-library-analytics-setup.js]: > Correct me if I'm wrong, but this isn't using the actual component-library AdditionalInfo component, but more of a static site / Drupal recreation?

you are correct!

**Pull request merged**